templates/python/custom-copilot-basic/src/bot.py.tpl

```diff
@@ -65,4 +62,8 @@ async def on_error(context: TurnContext, error: Exception):
     traceback.print_exc()
 
     # Send a message to the user
-    await context.send_activity("The bot encountered an error or bug.")
\ No newline at end of file
+    await context.send_activity("The bot encountered an error or bug.")
+
+@bot_app.feedback_loop()
+async def feedback_loop(_context: TurnContext, _state: TurnState, feedback_loop_data: FeedbackLoopData):
+    print(feedback_loop_data)
```

Comment:
Please add some comments like: Add your custom logic to process user feedback here.

Reply:
ok

Reply:
Will the feedback loop data be serialized first when printing?